lib/themeing/scss_preprocessor.rb

```diff
@@ -37,7 +37,7 @@ def original_scss
 
     def replace_top_comment(scss)
       lines = scss.split("\n")
-      loop { lines[0].match?(%r{\A(//|\w*\z)}) ? lines.shift : break }
+      loop { lines[0].match(%r{\A(//|\w*\z)}) ? lines.shift : break }
```

Comment:
I am mystified by this. `match` was updated to `match?` between ruby 2.3.x and ruby 2.4.x. But this errored when running with `bundle exec` with ruby at 2.4. Is there any explanation for that?

Reply:
turns out there is - bundler was using 2.3.x